app/views/state_file/questions/nj_review/edit.html.erb

```diff
@@ -46,5 +46,16 @@
     </div>
   <% end %>
 
+  <div id="rent_paid" class="white-group">
```

Comment:
[pebble] copy paste error "rent paid"

Reply:
Thanks for the catch. Any idea why we have ids on these anyway?